src/main.rs

```diff
@@ -1,9 +1,22 @@
 use geo::Point;
+use serde_json;
+use serde_json::{to_string_pretty};
 use zonebuilder::clockboard;
 use zonebuilder::Params;
 
 fn main() {
-    let polygon_list = clockboard(Point::new(0.0, 0.0), Params::default(), None);
-    let geojson_list = geojson::Value::from(&polygon_list[0]);
-    println!("{}", geojson_list);
+    let gj = clockboard(Point::new(0.0, 0.0), Params::default());
+
+    // Attempt to print pretty json - not outputting valid json currently
+    // See https://github.com/georust/geojson/issues/161 for details
+    // let geojson_list = geojson::Value::from(&polygon_list[0]);
+    // let result = serde_json::to_string_pretty(&geojson_list);
+    // println!("{}", result.unwrap());
+
+    let gjstring = to_string_pretty(&gj);
```

Comment:
It's more typical to do the `unwrap()` here

Reply:
You mean:

```
     let gjstring = to_string_pretty(&gj).unwrap();
```

?

Reply:
Correct. The idea is that `to_string_pretty` [might fail](https://docs.serde.rs/serde_json/fn.to_string_pretty.html#errors). There are [many ways to do error handling](https://doc.rust-lang.org/book/ch09-00-error-handling.html), but in this case, you just want to halt and dump the error if something goes wrong -- that's what `unwrap` does. Since you want to later use the `String` and forget about the fact that there might've been an error producing it, it makes the flow of logic simpler to do the `unwrap` immediately. Let's say you used the string several times below -- repeatedly checking for errors and crashing if so is harder to reason about.